src/Vehicle/Vehicle.cc

```diff
@@ -1566,6 +1571,82 @@ void Vehicle::_handleWind(mavlink_message_t& message)
 }
 #endif
 
+void Vehicle::_handleServoOutputRaw(const mavlink_message_t& message)
+{
+    if(_gripperNumber > 0 && _gripRelease >= 1000 && _gripGrab >= 1000) {
```

Comment:
These values are too strict, use `>=900` (Malloy's release is at 982).

Reply:
@almatthew main reason I put that is because I saw from https://ardupilot.org/copter/docs/parameters.html#grip-release-gripper-release-pwm that the range is 1000-2000. Is that going to be an issue with this?

Reply:
You can ignore that; there is no (AFAIK) limit on those upper/lower bounds, and servo outputs are very often <1000 and > 2000